lib/awful/hotkeys_popup/keys/tmux.lua

```diff
@@ -0,0 +1,73 @@
+---------------------------------------------------------------------------
+--- tmux hotkeys for awful.hotkeys_widget
+--
+-- @autor nahsi nashi@airmail.cc
+-- @copyright 2017 nahsi
+-- @module awful.hotkeys_popup.keys.tmux
+---------------------------------------------------------------------------
+
+local hotkeys_popup = require("awful.hotkeys_popup.widget")
+
+local tmux_rule = { name = "tmux" }
```

Comment:
> but it's never like that even with default tmux config
Really? I copied that from vim.lua hotkeyfile. (i.e it's not a problem there). Also I have a default 'tmux' config and 'termite' terminal, 'tmux' always shows up in a title. Tried xfce4-terminal also and 'tmux' showed up. 

```
if tmux_rule == nil then
    t_rule = { name = "tmux" }
else
    t_rule = tmux_rule
end
for group_name, group_data in pairs({
    ["tmux: sessions"] = { rule = t_rule },
    ["tmux: windows"]  = { rule = t_rule },
    ["tmux: panes"]    = { rule = t_rule },
    ["tmux: misc"]     = { rule = t_rule },
}) do
    hotkeys_popup.add_group_rules(group_name, group_data)
end

```

Is this acceptable? The user needs to define  variable with rule, if no variable defined then `name = "tmux" is used`.